docs/markdown/Introduction/sponsorship.md

```diff
@@ -0,0 +1,57 @@
+
+We offer formal sponsorship tiers for companies, as well as individual sponsorships via [GitHub](https://github.com/sponsors/pantsbuild).
+
+We are very grateful to our sponsors, whose support allows us to continue to maintain and release Pants!
+
+Corporate Sponsorship
+=====================
+
+Corporate sponsorships are formal agreements with the Pants Build organization. Sponsors receive regular access to project maintainers, privileged support channels, and listings and acknowledgements on our website.
+
+The corporate sponsorship tiers are:
+
+|                                    | Platinum  | Gold           | Silver     |
+|------------------------------------|-----------|----------------|------------|
+| Fee, if paid anually               | $5,000    | $3,000         | $1,500     |
+| Fee, if paid monthly               | $500      | $300           | $150       |
+| Maximum number of sponsors         | 2         | 5              | Unlimited  |
+| 60 min consultation calls          | 1/quarter | 1/year         | -          |
+| Privileged support channel         | yes       | yes            | yes        |
+| Jobs board placement               | Highest   | Below Platinum | Below Gold |
+| Acknowledgement in release notes   | yes       | -              | -          |
+| Placement on website sponsors list | Highest   | Below Platinum | Below Gold |
+| Informational text on website      | 300 words | 200 words      | 100 words  |
+
+Additional tiers are available by negotiation for contributions above $5,000, or for in-kind contributions of essential infrastructure or personnel by agreement with the Pants Build board.
+
+## Consultation calls with maintainers
+Platinum and Gold sponsors are entitled to a regular consultation with a Pants Maintainer. This may be in the form of a 1:1 discussion with a maintainer, or a tech talk with your engineering department.
+
+## Privileged support channel
+A selection of Pants Maintainers are willing to support in a confidential channel, including entering into a confidentiality agreement, allowing greater insight when troubleshooting issues with your Pants environment.
+
+## Jobs Board placement
+The Pants [jobs board](https://www.pantsbuild.org/page/jobs) is an online resource for posting job opportunities relevant to experienced build engineers. Postings here are visible to Pants Maintainers and Contributors, as well as our user base. Sponsors will receive more prominent placement on our jobs board.
+
+## Acknowledgement in release notes
+Platinum sponsors will be acknowledged as a financial contributor to each Pants release series made during their sponsorship term, in blog announcements and the release notes included with each distribution.
+
+## Acknowledgement on the Pants website
+The [Pants website](https://www.pantsbuild.org/) will include a sponsor section, displaying logos and informational text about each sponsor. Sponsors will be sorted according to tier.
```

Comment:
And for those on the same tier? alphabetically perhaps? or by date of becoming sponsors?

Reply:
Generally speaking, it's best not to constrain ourselves unnecessarily.

Reply:
I'm not against not mentioning sorting at all, just felt odd to only specify the sorting half-ways.

Reply:
I think we left it deliberately open so we have some wiggle room. This is a marketing document, not pseudocode :)